STYLE-GUIDE.md

```diff
@@ -0,0 +1,22 @@
+# Exercism's Style Guide
+
+This document acts as a Style Guide for the language and wording used in exercises.
+
+## General principle
+
+All content should be written in US english. 
+In the future other translations may occur, but the "official" Exercism natural langauge is US English.
+
+## Explain or substitude mathmatical and esotoric terms.
```

Comment:
Could you perhaps add an example of a non-maths esoteric term? The content of this section only mentions maths terms.

Reply:
I can't think of one off the top of my head. But I'll try and come up with one and PR it. Feel free to do the same if you can think of one.

Reply:
I would guess that if I were talking about "lower case" it would be the beer that is lower than the other case of beer.  While saying "lowercase" is talking about letters.